### PR TITLE
Feature/plcn 12 supplier info summary

### PR DESCRIPTION
**Pull request by itm-mne**

### CODE REVIEW

File: src/components/summaries/SupplierInfoSummary.tsx [hunk 1]
```diff
@@ -4,99 +4,117 @@ import React, { FC } from 'react';
...
+							</Box>
+						</Tooltip>
+					))}
+					<Typography variant="body1" fontWeight={600} sx={{ opacity: 0.7 }}>
+						+{supplierArray.length - 2}
+					</Typography>
+				</>
+			);
+		} else {
+			console.log('Der er noget rygende galt');
```
Comment (LarsITMinds) [inline src/components/summaries/SupplierInfoSummary.tsx:101]: I guess this is a scenario that will 'never' happen.
If so, still try to handle the error the best way possible, with a proper error message :)

File: src/pages/WallOfClients.tsx [hunk 2]
```diff
@@ -9,10 +8,10 @@ import PageContainer from '../components/common/PageContainer';
 import Underlined from '../components/common/Underlined';
 import ClientInfoSummary from '../components/summaries/ClientInfoSummary';
 import DealsStatusSummary from '../components/summaries/DealsStatusSummary';
-import { DealStatus } from '../components/summaries/DealsStatusSummary';
 import SupplierInfoSummary from '../components/summaries/SupplierInfoSummary';
-import { SupplierSummary } from '../components/summaries/SupplierInfoSummary';
 import { flexCenter, flexCol } from '../styles/generalStyles';
+// Dummy classes for testing
+import { dummySuppliers4 } from '../utils/dummyClasses';
```
Comment (LarsITMinds) [inline src/pages/WallOfClients.tsx:14]: If this is getting removed later, I will suggest putting in a TODO in the comment, so it is easier to search through the project later for stuff that needs to be deleted or changed. Stuff like this tends to get forgotten :)

**Review by LarsITMinds — CHANGES_REQUESTED** [review]

File: src/components/summaries/SupplierInfoSummary.tsx (hunk 1 above)
Comment (s-bogelund) [reply src/components/summaries/SupplierInfoSummary.tsx:101]: Created an issue in Jira to fix this

File: src/components/summaries/SupplierInfoSummary.tsx [hunk 3]
```diff
@@ -4,99 +4,117 @@ import React, { FC } from 'react';
...
+							<Box
+								width="70%"
+								display="flex"
+								flexDirection="column"
+								alignItems="start"
+								gap="0"
+								mt="2px"
+							>
+								<Box width="100%" height="fit-content" maxHeight="22px">
+									<Typography variant="body1" noWrap>
```
[the remark this answers is not in the document]
Comment (s-bogelund) [reply src/components/summaries/SupplierInfoSummary.tsx:44]: Created an issue in Jira to address this

**Pull request merged**